simple-auth-server/src/email_service.rs

```diff
@@ -0,0 +1,71 @@
+use models::Invitation;
+use sparkpost::transmission::{
+    EmailAddress, Message, Options, Recipient, Transmission, TransmissionResponse,
+};
+
+fn get_api_key() -> String {
+    std::env::var("SPARKPOST_API_KEY").expect("SPARKPOST_API_KEY must be set")
+}
+
+#[allow(unused)]
```

Comment:
What is unused here? If you want disable such warnings, do not assign variable or use `let _`

Reply:
done